django_scim/views.py

```diff
@@ -38,9 +38,33 @@
 
 
 class SCIMView(View):
+    lookup_field = 'id'
+    lookup_url_kwarg = 'uuid'
 
     implemented = True
 
+    def get_object(self):
+        """Get object by configurable ID."""
+        # Perform the lookup filtering.
+        lookup_url_kwarg = self.lookup_url_kwarg or self.lookup_field
+
+        uuid = self.kwargs[lookup_url_kwarg]
```

Comment:
Should we use `.get()` here to avoid a `KeyError`?

Reply:
Good call, will submit a fix shortly.

Reply:
Actually no, sorry. The key must be present, there's an assert for this a few lines down. This logic was more or less copied from the way django rest_framework allows you to configure your URLs.

If that raised a KeyError it would mean that either your URLs are misconfigured or lookup_field is defined incorrectly, it's not something that can be triggered by an end-user in a production environment.

Reply:
I'll fix the assert to come first since it gives a nicer error message than KeyError ;-)